rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -50,6 +50,9 @@ def testBadAtomHandling(self):
       for nm, fn in Descriptors._descList:
         try:
           v = fn(m)
+        except RuntimeError:
+          # 3D descriptors fail since the mol has no conformers
+          pass
```

Comment:
Same here. This was included to quiet the test for the 3D descriptors. As we removed them, this exception handling is no longer required